build/build.sh

```diff
@@ -29,3 +29,7 @@ goflags=${GOFLAGS:-}
 
 
 CGO_ENABLED=0 GO111MODULE=on GOOS=linux go build ${GOFLAGS} -ldflags="-X 'github.com/openshift/windows-machine-config-operator/version.Version=${VERSION}'" -o ${BIN_DIR}/${BIN_NAME} ${MAIN_PACKAGE}
+
+# Build promu utility tool from vendor dir. Promu is used for building prometheus projects like windows_exporter
+echo "building promu..."
+CGO_ENABLED=0 GO111MODULE=on GOOS=linux go build ${GOFLAGS} -o ${BIN_DIR}/promu "github.com/prometheus/promu"
```

Comment:
Shouldn't we be using `github.com/openshift/prometheus-promu`?

Reply:
That'd have to go into go.mod as a replace directive. Is there any reason for creating the fork other than needing it for promu as a submodule? If not, I suggest we drop maintenance of the fork entirely. If there are however, I'll change this. Please advise.

Reply:
That's a good point @LorbusChris. We should no longer need the fork.

Reply:
I am not 100% convinced  on doing the build here. Wouldn't it be better to do the build as part of the container build? Because I could make the same argument about building everything else (kubelet, kube-proxy etc) here also.

Reply:
Main reason for adding it here was that we only copy the repo (and with it the vendor dir) over to the container image in the CI Dockerfile.
I think the best fix would indeed be to _always_ build everything within a container though, even in the non-CI Dockerfile (currently that copies the already built binaries into the container which isn't ideal, in case one forgets to run `make build` before...)

Reply:
>I think the best fix would indeed be to always build everything within a container though, even in the non-CI Dockerfile

We are going to do that when we move to SDK 1.0. So I suggest we move to building promu in the Dockerfile also.